src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/TypePreinit.cs

```diff
@@ -707,6 +759,7 @@ private Status TryScanMethod(MethodIL methodIL, Value[] parameters, Stack<Method
                     case ILOpcode.conv_i4:
                     case ILOpcode.conv_i8:
                     case ILOpcode.conv_u2:
+                    case ILOpcode.conv_u4:
```

Comment:
It looks like that places like this one will end up handling all integer types in the fullness of time. Would it make sense to add them all and be done with it instead of adding one at a time?

(IIRC, we have discussed in one of the earlier preinit PRs.)